config/clowder/bases/clowdenv.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+# Custom Resource defined as part of the Clowder API
+kind: ClowdEnvironment
+metadata:
+  name: env-default
+spec:
+  targetNamespace: default
+
+  # Providers all your app to consume configuration
+  # data automatically based on your request
+  providers:
+
+    # provides a k8s service on port 8000
+    web:
+      port: 8000
+      privatePort: 8080
+      mode: operator
+
+    # provides a prometheus metrics port on 9000
+    metrics:
+      port: 9000
+      mode: operator
+      path: "/metrics"
+
+    # Creates a kafka pod in the targetNamespace
+    kafka:
+      namespace: default
+      clusterName: crc-cluster
+      mode: local
```

Comment:
Are these actually needed? Maybe the metrics yes for clowder?

Reply:
Set to `none`. Thanks!